front/package.json

```diff
@@ -1,75 +1,76 @@
 {
-  "name": "front",
-  "version": "0.1.0",
-  "private": true,
-  "scripts": {
-    "local": "next dev",
-    "dev_sst": "bash ./scripts/yarn_dev_sst.sh",
-    "dev": "bash ./scripts/yarn_dev.sh",
-    "deploy": "bash ./scripts/yarn_deploy.sh",
-    "teardown": "bash ./scripts/yarn_teardown.sh",
-    "build": "next build",
-    "start": "next start",
-    "lint": "next lint",
-    "test": "jest",
-    "test:watch": "jest --watch",
-    "generate": "yarn prisma generate --generator client_js",
-    "db-reset": "yarn prisma db push --force-reset --skip-generate",
-    "seed": "yarn prisma db seed",
-    "format": "prettier --write ."
-  },
-  "prisma": {
-    "schema": "../prisma/schema.prisma",
-    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} ./prisma/seed.ts"
-  },
-  "engines": {
-    "node": ">=18.0.0"
-  },
-  "dependencies": {
-    "@auth/prisma-adapter": "^1.0.1",
-    "@emotion/react": "^11.11.1",
-    "@emotion/styled": "^11.11.0",
-    "@fortawesome/fontawesome-svg-core": "^6.4.2",
-    "@fortawesome/free-solid-svg-icons": "^6.4.2",
-    "@fortawesome/react-fontawesome": "^0.2.0",
-    "@headlessui/react": "^1.7.17",
-    "@mui/material": "^5.14.8",
-    "@mui/x-date-pickers": "^6.12.1",
-    "@prisma/client": "^5.2.0",
-    "@radix-ui/react-tooltip": "^1.0.6",
-    "@types/node": "^20.5.7",
-    "@types/react": "18.2.18",
-    "@types/react-dom": "18.2.7",
-    "autoprefixer": "10.4.14",
-    "bcryptjs": "^2.4.3",
-    "dayjs": "^1.11.9",
-    "eslint": "8.46.0",
-    "eslint-config-next": "13.4.13",
-    "next": "13.4.12",
-    "next-auth": "^4.23.1",
-    "postcss": "8.4.27",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "server-only": "^0.0.1",
-    "tailwindcss": "3.3.3",
-    "tailwindcss-group-modifier-plugin": "^0.1.1",
-    "typescript": "5.1.6"
-  },
-  "devDependencies": {
-    "@testing-library/jest-dom": "^6.0.1",
-    "@testing-library/react": "^14.0.0",
-    "@types/bcryptjs": "^2.4.3",
-    "@types/jest": "^29.5.3",
-    "@typescript-eslint/eslint-plugin": "^6.3.0",
-    "aws-cdk-lib": "2.84.0",
-    "constructs": "10.1.156",
-    "eslint-config-prettier": "^9.0.0",
-    "eslint-plugin-prettier": "^5.0.0",
-    "jest": "^29.6.2",
-    "jest-environment-jsdom": "^29.6.2",
-    "prettier": "^3.0.1",
-    "prisma": "^5.2.0",
-    "sst": "^2.23.6",
-    "ts-node": "^10.9.1"
-  }
+    "name": "front",
```

Comment:
Hmm 🤔 Why did this entire file change? Looks like you might have a formatting setting in vscode that isnt the same as the rest of the team?

Reply:
My VSCode does not reformat the package.json to its old format. I guess that the setting is not defined. Using 4 spaces for a tab is now consistent with prettier.